history/characters/29000_gnoll.txt

```diff
@@ -1519,4 +1488,66 @@
 	add_trait=intricate_webweaver
 	580.3.18={ birth=yes add_trait=creature_gnoll }
 	630.12.7={ death=yes }
-}
\ No newline at end of file
+}
+#dynasty=12236
+29194={
+	name=Razer
+	dynasty=12236
+	culture=gnoll religion=primitive_shamanism
+	martial=9 diplomacy=4 stewardship=6 intrigue=9 learning=2
+	trait=stutter
+	disallow_random_traits = yes
+	472.03.06={ birth=yes add_trait=creature_gnoll}
+	488.03.06 = {
+	add_trait=flamboyant_schemer
+	add_trait=deceitful
+	add_trait=envious
+	add_trait=cynical
+	}
+	509.06.26 = {
+		death = {
+			death_reason = death_battle killer = 1
+		}
+	}
+}
+29205={
+	name=Shredder
+	dynasty=12237
+	culture=gnoll religion=primitive_shamanism
+	martial=2 diplomacy=6 stewardship=11 intrigue=4 learning=4
+	trait=weak
+	disallow_random_traits = yes
+	453.07.18={ birth=yes add_trait=creature_gnoll}
+	469.07.18 = {
+	add_trait=fortune_builder
+	add_trait=paranoid
+	add_trait=just
+	add_trait=stubborn
+	}
+	519.03.09 = {
+		death = {
+			death_reason = death_poor_health
+		}
+	}
+}
+29206={
+	name=Teef
+	dynasty=12001
+	culture=gnoll religion=primitive_shamanism
+	martial=9 diplomacy=0 stewardship=1 intrigue=3 learning=0
```

Comment:
Why some of his stats are 0? Try balancing his stats around the total number of 24.

Reply:
done